src/RegExpr.pas

```diff
@@ -2679,19 +2672,24 @@ function TRegExpr.ParsePiece(var flagp: integer): PRegExprChar;
     if regcode <> @regdummy then
     begin
       PREBracesArg(AlignToInt(Result + REOpSz + RENextOffSz))^ := ABracesMin;
-      PREBracesArg(AlignToInt(Result + REOpSz + RENextOffSz + REBracesArgSz))^
-        := ABracesMax;
+      PREBracesArg(AlignToInt(Result + REOpSz + RENextOffSz + REBracesArgSz))^ := ABracesMax;
     end;
   end;
 
+var
+  op: REChar;
+  NonGreedyOp, NonGreedyCh: boolean; // ###0.940
```

Comment:
Sorry I do not get it - somewhere it starts from capital letter, somewhere else this is cameCase.

Reply:
NonGreedyOp / NonGreedyCh are your old names, i moved them lower to make them not visible by included funcs.